### DISCUSSION TITLE
Crash in conhost?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#15826] [General]
Any ideas on why conhost might crash (Access violation - code c0000005 (!!! second chance !!!)), at ntdll!RtlCustomCPToUnicodeN+0xa9?

**Comment by DHowett** [comment]
This is a bug in conhostv1 (the “legacy” console) that resulted from changes to the PEB to accommodate per-process ACP. This should only occur on 22000+ (“Windows 11”).

Notes here: https://github.com/microsoft/terminal/issues/15038

Fixed, but I can’t give a release timeline.

Viable workaround: stop using conhostv1; it’s 2023.

**Comment by vefatica** [comment]
Here's a [link](https://jpsoft.com/forums/threads/crash-in-conhost-exe.11639/) to the original report.  It's from a JPSoft/TCC user who says he's using Win-10, version 22H2 (OS-build 19045.3324).

Are you saying it won't happen with the legacy console in Windows 10?  If so, I reckon it's something else.

**Reply by zadjii-msft** [reply]
MSFT:42362468 points at MSFT:41399222 which points at [KB5027303](https://support.microsoft.com/en-us/topic/june-27-2023-kb5027303-os-build-22621-1928-preview-1ada2c0a-fa85-43f8-91c4-6ee13fdf278b), which is OS Build 22621.1928. 

I believe that regressed in OS!4161028, in Dec 2019... that might have made its way to Win10 builds. Hard to say...
The revert was in os.2020!8161820 in Dec 2022. Interesting.

**Comment by vefatica** [comment]
I guess that was it.  The user has reported 

> Now after switching off Lecacy Mode, I've not seen the same crash again.

Thanks!
